opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -641,7 +638,7 @@ export const useBuildFilterKeysMapFromEntityType = (entityTypes = ['Stix-Core-Ob
   // 2. case several entity types
   const filterKeysMap = new Map();
   entityTypes.forEach((entityType) => {
-    const currentMap = filterKeysSchema.get(entityType);
+    const currentMap = filterKeysSchema.get(entityType) ?? new Map();
     currentMap?.forEach((value, key) => {
```

Comment:
useless '?' if you set the default value to new Map() the line juste above

Reply:
Yes, let me remove it right now